inst/schema/report_run_request.json

```
@@ -0,0 +1,27 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string"
+        },
+        "branch": {
+            "type": "string"
+        },
+        "hash": {
+            "type": "string"
+        },
+        "parameters": {
+            "oneOf": [
+                { "type": "null" },
+                {
+                    "type": "object",
+                    "properties": {},
+                    "additionalProperties": true
```

```suggestion
                    "additionalProperties": {
                      "type": ["boolean", "integer", "number", "string"]
                    }
```

we can be stricter here, and the `properties` key does nothing in the current version

---

done